### PR TITLE
Fix for Multiplayer

### PR DESCRIPTION
**Pull request by schwiti6190**

### CODE REVIEW

**Comment by schwiti6190** [comment]
https://github.com/Courseplay/courseplay/issues/4711#issue-542963478

File: CombineAIDriver.lua [hunk 1]
```diff
@@ -97,8 +97,10 @@ function CombineAIDriver:init(vehicle)
 				self.pipe:setAnimationTime(self.pipe.animation.name, 1, true)
 			else
 				-- if there's no animation we have to use this, as seen in the Giants pipe code
-				self.pipe:setPipeState(CombineAIDriver.PIPE_STATE_OPEN)
-				self.pipe:updatePipeNodes(999999, nil)
+				--self.pipe:setPipeState(CombineAIDriver.PIPE_STATE_OPEN)
```
Comment (pvaiko) [inline CombineAIDriver.lua]: Please don't comment out  unused code, remove it.

Comment (schwiti6190) [reply CombineAIDriver.lua]: I deleted the old code

**Comment by pvaiko** [comment]
Thanks for the contribution!

**Pull request merged**